src/components/LocationDetails.js

```diff
@@ -30,15 +30,31 @@ const LocationDetails = (props) => {
           </ReactTooltip>
         )}
         {comparisonFts.length ? (
-          allFeatures.map(ft => 
-            <MetricDetails key={ft.properties.SA2_MAIN16} feature={ft} metric={metric} />
-          )
+          <MetricDetails featureList={allFeatures} metric={metric} />
         ) : (
-          <p>{value}</p>
+          renderSelectedValue(metric)
         )}
       </div>
     )
   }
+  
+  const renderSelectedValue = (metric) => {
+    let rawValue = selectedFeature.properties[metric.id];
+    const value = formatValue(rawValue, metric.format);
+    
+    switch (metric.type) {
+      case 'line-chart': {
+        const data = JSON.parse(rawValue);
```

Comment:
This crashes with `SyntaxError: JSON Parse error: Unexpected identifier "object"` if I search for anything when there's nothing in `comparisonFeatures`.